Adding a code sample of how the tabs widget can be initialized and used. (#4242)
magento/devdocs: guides/v2.1/javascript-dev-guide/widgets/widget_tabs.md

### `dimensionsChanged` {#fedg_tabs_dimensionsChanged}
Called after content is opened or closed.

> ## Code Sample

Reviewer: ```suggestion
## Code sample
```

> ## Code Sample {#tabs_code_sample}

Reviewer 2: Please, remove `{#tabs_code_sample}` part. As it's preferable to use a clean Markdown, the links to the sections can be processed using Markdown.

The following example shows how to initialize the tabs widget and pass options during the initialization.

> The classes used are only for getting the same view as the tabs have it on product page.

Reviewer: ```suggestion
This example uses the same CSS classes as the tabs on the product page.
```

```html
<div class="product data items"
     data-mage-init='{"mage/tabs": {"openedState": "active", "animate": {"duration": 100}, "active": 1, "disabled": [2], "disabledState": "disabled"}}'>
    <div class="item title" data-role="collapsible">
        <a class="switch" data-toggle="trigger" href="#tab-cars">Cars</a>
    </div>
    <div id="tab-cars" class="item content" data-role="content">Cars content</div>
    
    <div class="item title" data-role="collapsible">
        <a class="switch" data-toggle="trigger" href="#tab-movies">Movies</a>
    </div>
    <div id="tab-movies" class="item content" data-role="content">Movies content</div>

    <div class="item title" data-role="collapsible">
        <a class="switch" data-toggle="trigger" href="#tab-music">Music</a>
    </div>
    <div id="tab-music" class="item content" data-role="content">Music Content</div>
</div>
```

### Result

> The result are 3 tabs with its content, where the last tab is disabled based on initialization options, as shown here:

Reviewer: ```suggestion
The result is three tabs with content, where the last tab is disabled based on initialization options, as shown here:
```

> The result are 3 tabs with its content, where the last tab is disabled based on our initialization options, as shown here:

Reviewer 2: Please, rephrase the "our initialization" part by removing "our". 
So, in result we have something like "tab is disabled based on the initialization options"
